### DISCUSSION TITLE
Pages are Created With 1h delay

### DISCUSSION THREAD

**Discussion by cleverson-aranda** [requarks/wiki#6685] [Help / Questions] [answered]
Good morning!

Please, does anyone know how I can correct the time it informs that the pages are created, when I create a page it is 1 hour late, I have already checked the container that is running the wiki and the database and both have the correct time

![page_test](https://github.com/requarks/wiki/assets/35637410/720bf31c-257d-4146-840b-89d1e056a3ad)

**Comment by P4mes4n** [comment, marked as answer]
Set the Timezone in your Wiki-Profile in the top right corner.

**Reply by cleverson-aranda** [reply]
Thank you so much! It's work
